GNUmakefile

```diff
@@ -40,8 +40,12 @@ run: ## Run the wallabag built-in server
 	@$(PHP) bin/console server:run --env=dev
 
 build: ## Run webpack
-	@$(YARN) install
-	@$(YARN) build:$(ENV)
+	$(YARN) install
+	@if [ "$(ENV)" = "prod" ]; then \
+		$(YARN) build; \
+	else \
+		$(YARN) dev; \
+	fi;
```

Comment:
to me here we should always use dev command, assuming the make file is more for dev purpose